content/docs/pulumi-cloud/get-started/onboarding-guide/migrating-to-pulumi.md

```diff
@@ -0,0 +1,79 @@
+
+## Migrating existing infrastructure
+
+If you have existing cloud infrastructure to bring into Pulumi IaC, you have several strategies available.
+
+### Choose your migration approach
+
+**Start fresh:** Simply throw away existing infrastructure and begin anew. This ensures you can adopt all best practices from the outset without technical debt. This option isn't always practical for business-critical services.
+
+**Import existing infrastructure:** Pulumi has tools to import any cloud infrastructure regardless of how it was created — even manually through cloud consoles. Pulumi offers tailored migration tools for Terraform, AWS CloudFormation/CDK, Azure ARM, and Kubernetes YAML. These tools generate Pulumi IaC code in your chosen language and actively place existing resource management under Pulumi IaC, swapping out management without disrupting resources for zero downtime.
+
+**Coexist and migrate incrementally:** Pulumi supports coexisting with existing ecosystems. You can deploy Helm charts as-is or consume Terraform workspace outputs. This enables incremental migration over time when the value is right.
+
+**Get professional help:** Pulumi offers professional services to help with migration.
+
+{{% notes type="info" %}}
+
+Learn more at the [Pulumi Migration Hub](https://www.pulumi.com/docs/iac/adopting-pulumi/migrating-to-pulumi/) or [detailed migration tooling documentation](https://www.pulumi.com/docs/iac/adopting-pulumi/).
+
+{{%/notes%}}
+
+## Drive and measure migration success
+
+Successfully migrating to Pulumi requires treating it like a product launch with clear success metrics and strategic execution.
+
+### Start with a beachhead win
+
+**Goal:** Get your first 1-3 workloads into production as soon as possible (typically 3-6 months for most organizations).
+
+This "beachhead" win accomplishes three things:
+
+- Keeps you grounded in reality without going dark too long
+- Lays groundwork for onboarding more workloads while improving your platform
+- Provides tangible accomplishments to showcase to management and broader teams
+
+These workloads should be automated with CI/CD pipelines and use as many best practices as possible, even though you'll be early in figuring out components and templates.
+
+### Stay focused on impact
+
+**Take a "workload-first" strategy:** Rather than creating dozens of abstractly-useful components, inform specific component requirements from real-world applications emerging from your beachhead win.
+
+**Resist the redesign temptation:** Don't conflate redesigning projects with new cloud architectures and platform migration. This adds risk. Get workloads onto Pulumi first, then refactor and redesign in place.
+
+### Treat your platform like a product
+
+An internal cloud platform is a product requiring superb developer experiences. While self-service is the primary goal, it's a journey. Start by:
+
+- Getting your platform well-architected
+- Documenting components and templates
```

Comment:
To the "Documengint components and templates" bullet item, add something like
"Use Pulumi Cloud IDP to provide user visibility and access to your templates and components.: